integration/token/common/topology.go

```diff
@@ -42,6 +42,7 @@ type Opts struct {
 	OnlyUnity           bool
 	FSCBasedEndorsement bool
 	ExtraTMSs           []TMSOpts
+	FinalityType        string
```

Comment:
We can use the type that exists already `finality.ManagerType` and the two constants `finality.Delivery` `finality.Committer`

Reply:
okay